prometheus/Dockerfile

```diff
@@ -17,11 +17,19 @@ RUN \
     && if [ "${BUILD_ARCH}" = "aarch64" ]; then ARCH="arm64"; fi \
     \
     && apk update \
-    && apk --no-cache add python3 py3-idna py3-certifi py3-chardet py3-yaml py3-urllib3 py3-requests \
-    && apk --no-cache add --virtual builddeps py-pip \
+    && apk --no-cache add \
+        python3 \
+        py3-idna \
+        py3-certifi \
+        py3-chardet \
+        py3-yaml \
+        py3-urllib3 \
+        py3-requests \
```

Comment:
Any reason why those are not in a requirement file?
We can leverage the HA pre-build wheels server in that case as well.

Reply:
i had some issues where pip didn't install all the requirements because pip has the same requirements, thats why

Moving them to a requirements file would be better